src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -33,6 +33,24 @@ public class QBittorrent : TorrentClientBase<QBittorrentSettings>
 
         private IQBittorrentProxy Proxy => _proxySelector.GetProxy(Settings);
 
+        public override void MarkItemAsImported(DownloadClientItem downloadClientItem)
+        {
+            // set post-import category
+            if (Settings.TvImportedCategory.IsNotNullOrWhiteSpace() &&
+                Settings.TvImportedCategory != Settings.TvCategory)
+            {
+                try
+                {
+                    Proxy.SetTorrentLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
+                }
+                catch (DownloadClientException ex)
+                {
+                    _logger.Warn(ex, "Failed to set torrent label \"{0}\" for {1} in qBittorrent",
```

Comment:
So I am trying to clean up some of the exceptions being dealt with, and I am thinking maybe this isn't too useful? We already have the exception handled in DownloadEventHub, and there doesn't seem to be a point in handling it earlier here.

Reply:
DownloadEventHub has no knowledge about what we were trying to do: setting the import category. So you'd lose that information if you didn't catch the exception here.